ansible/Vagrantfile

```diff
@@ -47,6 +48,13 @@ Vagrant.configure(2) do |config|
   end
 
   [
+    "omeroscriptsdependencies-debian"
+  ].each do |server|
+    config.vm.define "#{server}" do |node|
+      node.vm.box = "ubuntu/trusty64"
+      node.vm.provision "ansible" do |ansible|
+        ansible.playbook = "test.yml"
```

Comment:
The merge has gone wrong, you'll need to add three `end`s in to close the three loops.